single_kernel_mongo/managers/config.py

```diff
@@ -190,30 +219,37 @@ def configure_and_restart(self):
                 raise
 
 
-class MongoConfigManager(CommonConfigManager, ABC):
+class MongoConfigManager(FileBasedConfigManager, ABC):
     """The common configuration manager for both MongoDB and Mongos."""
 
     auth: bool
 
     @override
     def build_parameters(self) -> list[list[str]]:
-        return [
```

Comment:
nit, add context about design decision:
"""<about this function>

We aim to pass most config options inside the config file instead of as parameters. In earlier charm versions we pass parameters, however for security reasons in LDAP params we have now decided to set these in the config file.
"""

Reply:
Indeed, very good idea!